client/providers/google.go

```diff
@@ -60,10 +77,16 @@ func (g *GoogleOp) RequestTokens(ctx context.Context, cicHash string) (*memguard
 		return uuid.New().String()
 	}
 
-	ch := make(chan []byte)
-	chErr := make(chan error)
+	ch := make(chan []byte, 1)
+	chErr := make(chan error, 1)
```

Comment:
I'm always a little wary when I see a change like this. I think you address this in a comment somewhere but is the purpose of the buffer to improve performance or to prevent a deadlock?

(I really hope it's the former!)

Reply:
This was a change requested by @ymarcus93 in his code review of this PR.

https://github.com/openpubkey/openpubkey/pull/68#discussion_r1444759044

I remember there being a much longer thread about this somewhere in the PR, but I can't seem to find it at the moment.  The short version is that the default in go if no size is specified is for a channel to have a buffer size of 0 elements. This makes the channel automatically block which could prevent the HTTP server from being cleaned up.  As a I understand it, the recommendation is to have channels of size 1 to prevent blocking behavior unless such blocks are an intended part of the design.